src/main/scala-2.11/markets/engines/CDAMatchingEngine.scala

```diff
@@ -82,6 +111,42 @@ class CDAMatchingEngine(askOrdering: PriceOrdering[AskOrder],
     math.min(incoming.quantity, existingOrder.quantity)
   }
 
+  /** Remove an order from the matching engine. */
+  def remove(existing: Order): Option[Order] = {
+    existing match {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1834547231/issues/source?bid=2921361&fileBranchId=3080901#l116)